configure/accessibility-options.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Accessibility options
+title_nav: Accessibility options
+description_short:
+description: Editor accessibility settings.
```

Comment:
This shouldn't say settings, though I see a lot still haven't been updated to replace that with options. It also needs to be a full sentence, so maybe something like this?
```suggestion
description: Configure the accessibility of TinyMCE's editable area.
```